From 9abef83897ed02be4d9cb55b8df0e403da65cd3a Mon Sep 17 00:00:00 2001
From: "F. von Gellhorn" <flinux@vongellhorn.ch>
Date: Sun, 16 Feb 2020 00:57:31 +0100
Subject: [PATCH 1/1] Solus tweaks to the default config

- Decrease desktop icon sizes
- Set default Monospace font size to 10
- Move showdesktop in front of the systemtray
---
 containments/desktop/package/contents/config/main.xml           | 2 +-
 kcms/fonts/fontssettings.kcfg                                   | 2 +-
 .../org.kde.plasma.desktop.defaultPanel/contents/layout.js      | 2 +-
 3 files changed, 3 insertions(+), 3 deletions(-)

diff --git a/containments/desktop/package/contents/config/main.xml b/containments/desktop/package/contents/config/main.xml
index 5703999..8c02e38 100644
--- a/containments/desktop/package/contents/config/main.xml
+++ b/containments/desktop/package/contents/config/main.xml
@@ -100,7 +100,7 @@
     </entry>
     <entry name="iconSize" type="Int">
       <label>The icon size to use for Folder View icons. One of the list of Plasma icon sizes.</label>
-      <default>4</default>
+      <default>3</default>
     </entry>
     <entry name="labelWidth" type="Int">
       <label>The width of the grid cells when using icon view. 0 = Narrow, 1 = Medium, 2 = Wide</label>
diff --git a/kcms/fonts/fontssettings.kcfg b/kcms/fonts/fontssettings.kcfg
index bd5ac71..652e76b 100644
--- a/kcms/fonts/fontssettings.kcfg
+++ b/kcms/fonts/fontssettings.kcfg
@@ -23,7 +23,7 @@
       #ifdef Q_OS_MAC
       QFont fixedWidthFont = QFont("Monaco", 10);
       #else
-      QFont fixedWidthFont = QFont("Hack", 9);
+      QFont fixedWidthFont = QFont("Hack", 10);
       fixedWidthFont.setStyleName("Regular");
       #endif
       </code>
diff --git a/layout-templates/org.kde.plasma.desktop.defaultPanel/contents/layout.js b/layout-templates/org.kde.plasma.desktop.defaultPanel/contents/layout.js
index 048671e..3d8dd49 100644
--- a/layout-templates/org.kde.plasma.desktop.defaultPanel/contents/layout.js
+++ b/layout-templates/org.kde.plasma.desktop.defaultPanel/contents/layout.js
@@ -78,7 +78,7 @@ if (langIds.indexOf(languageId) != -1) {
     panel.addWidget("org.kde.plasma.kimpanel");
 }
 
+panel.addWidget("org.kde.plasma.showdesktop")
 panel.addWidget("org.kde.plasma.systemtray")
 panel.addWidget("org.kde.plasma.digitalclock")
-panel.addWidget("org.kde.plasma.showdesktop")
 
-- 
2.25.0

